default/AI/AIDependencies.py

```diff
@@ -26,6 +26,8 @@
                   fo.planetSize.gasGiant: -1
                   }
 
+SUPPLY_MOD_SPECIALS = {'WORLDTREE_SPECIAL':{-1:1}}
```

Comment:
Find other example `-1` should not be mixed with other values in same inner dict.